hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -387,6 +387,12 @@ private FlinkOptions() {
       .noDefaultValue()
       .withDescription("Parallelism of tasks that do bucket assign, default is the parallelism of the execution environment");
 
+  public static final ConfigOption<Integer> HOODIE_ROECORD_MAP_TASKS = ConfigOptions
```

Comment:
typo ROECORD -> RECORD

Reply:
@wangxianghu 
thanks for review， i had fixed the problem what you pointed out
pls review again